docs/development/airflow.md

```diff
@@ -89,13 +91,13 @@ More information on running Airflow with Docker Compose can be found in the [off
 
 1. **Increase Airflow concurrency**. Modify the `docker-compose.yaml` and add the following to the x-airflow-common → environment section:
 
-```yaml
-AIRFLOW__CELERY__WORKER_CONCURRENCY: 32
-AIRFLOW__CORE__PARALLELISM: 32
-AIRFLOW__CORE__MAX_ACTIVE_TASKS_PER_DAG: 32
-AIRFLOW__SCHEDULER__MAX_TIS_PER_QUERY: 16
-AIRFLOW__CORE__MAX_ACTIVE_RUNS_PER_DAG: 1
-```
+    ```yaml
```

Comment:
None of this will work now because we are not using CeleryExecutor anymore.

Reply:
Oh, you are right. We are moving so fast, I totally forgot that we god rid of CeleryExecutor in https://github.com/opentargets/genetics_etl_python/pull/201.

However, only the first parameter (`AIRFLOW__CELERY__WORKER_CONCURRENCY`) was actually specific to CeleryExecutor. All of the rest ones apply to Airflow core or scheduler components which are independent from the executor.

I've now clarified this section in https://github.com/opentargets/genetics_etl_python/pull/221/commits/c17fbd239b88bf8515ed629ba88605a315566537.